src/Autodiscovery/AutodiscoveryLockServiceProvider.php

```diff
@@ -4,11 +4,21 @@
 
 use Goedemiddag\AutodiscoveryLock\Commands\AutodiscoveryPackageLock;
 use Goedemiddag\AutodiscoveryLock\Commands\AutodiscoveryPackageLockVerify;
+use Illuminate\Support\ServiceProvider;
 
-class AutodiscoveryLockServiceProvider extends \Illuminate\Support\ServiceProvider
+class AutodiscoveryLockServiceProvider extends ServiceProvider
 {
+    public function register(): void
+    {
+        $configPath = __DIR__ . '/../../config/autodiscovery.php';
```

Comment:
Since the variable `$configPath` is defined twice in the same manner, I think it could be moved to a `const`. Eg. `    protected const CONFIG_PATH = __DIR__ .  '/../../config/autodiscovery.php';
`

Reply:
Thanks for the feedback! I've changed it in commit https://github.com/goedemiddag/laravel-autodiscovery-lock/pull/28/commits/fbf9279f438106935f80ef27aeaafdd6c1a8a29c